x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { useFetchActiveMaintenanceWindows } from './use_fetch_active_maintenance_windows';
+import * as i18n from './translations';
+
+export const MaintenanceWindowCallout = () => {
```

Comment:
nit: It's absolutely optional but another options is to have a `function` on top level and specify a return type so the component as it improves readability and will look like

```
export function MaintenanceWindowCallout(): JSX.Element {
  ...
}
```